hplsql/src/main/java/org/apache/hive/hplsql/Var.java

```diff
@@ -612,7 +613,7 @@ else if (type == Type.TIMESTAMP) {
       if (t.length() > len) {
         t = t.substring(0, len);
       }
-      return String.format("TIMESTAMP '%s'", t);
+      return t;
```

Comment:
Is it safe to remove the `TIMESTAMP`?
A few lines above a comment says the value can be `.0`. Is it a float or a timestamp?

https://github.com/apache/hive/blob/be857bf83c00e41a315a7cc8e013832cf169ef18/hplsql/src/main/java/org/apache/hive/hplsql/Var.java#L608C54-L608C56

Reply:
As explained in https://github.com/apache/hive/pull/5182#discussion_r1570337451 comment, this was added by me only just reverted back the change done as a part of [HIVE-27492](https://issues.apache.org/jira/browse/HIVE-27492) fix.